packages/terra-tabs/src/common-tabs/CommonTabItem.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import TabContext from './_TabContext';
+
+const propTypes = {
+  /**
+   * Key to match with the activeItemKey to handle the display of selection.
+   */
+  itemKey: PropTypes.string.isRequired,
+  /**
+   * Text to be displayed on the tab or as it's aria-label.
+   */
+  label: PropTypes.string.isRequired,
+  /**
+   * Object to be returned in the onRequestActive.
+   */
+  metaData: PropTypes.object,
+  /**
+   * Function executed during rendering to generate child content.
+   */
+  render: PropTypes.func,
```

Comment:
should this be private prop ?

Reply:
This prop can be retained as it is, since it is expected to be passed down through the CommonTabs API. But won't be visible for framework variant. Current props table only display framework variant props